src/index.css

```diff
@@ -91,25 +112,35 @@ body {
   font-family: Poppins, sans-serif;
   font-weight: bold;
   font-size: 30px;
+  color: white;
   margin-right: 20px;
 }
 
 .reload {
   width: 40px;
   height: 40px;
+  background-color: aqua;
   margin-left: 20px;
   border-radius: 50px;
 }
 
+.reload:hover {
+  background-color: #fff;
+  color: rgb(232, 103, 123);
+  border: 3px solid rgb(232, 103, 123);
+}
+
 .context {
+  border: 4px solid rgb(255, 255, 255);
   width: 330px;
   overflow: scroll;
-  height: 55vh;
+  height: 450px;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding-top: 40px;
+  margin-bottom: 20px;
+  background-color: rgb(241, 172, 178);
 }
```

Comment:
- [ ] Kindly app padding-top and bottom of this wrapper to have the first and last player's name not hidden as in the screenshot below:

![image](https://user-images.githubusercontent.com/50941074/189321198-978fd88a-1098-41bd-a53b-3aff797b77b3.png)

- [ ] I tried this one and things were fine. `padding: 20px 10px;`